package.json

```diff
@@ -35,6 +35,7 @@
     "react-intl": "^2.1.2",
     "react-router": "^2.4.1",
     "react-treeview": "^0.4.5",
+    "remove-accents": "^0.4.0",
```

Comment:
Introducing a new dependency is always a source of complexity. 

I don't really know whether this is a good trade off in our case. Manually taking into account all the cases seems annoying, but the library doesn't seem to be standard either.

@MattiSG what do you think ?